pkg/contexts/ocm/transfer/transfer.go

```diff
@@ -116,7 +117,10 @@ func transferVersion(printer common.Printer, log logging.Logger, state WalkingSt
 	return list.Add(comp.AddVersion(t)).Result()
 }
 
-func CopyVersion(printer common.Printer, log logging.Logger, hist common.History, src ocm.ComponentVersionAccess, t ocm.ComponentVersionAccess, handler transferhandler.TransferHandler) error {
+func CopyVersion(printer common.Printer, log logging.Logger, hist common.History, src ocm.ComponentVersionAccess, t ocm.ComponentVersionAccess, handler transferhandler.TransferHandler) (rerr error) {
+	var finalize finalizer.Finalizer
```

Comment:
I don't understand this complexity. If you want it simpler then why isn't it enough to just plain call `m.Closer()`?